### ISSUE TITLE
Initialize device only when it is first discovered

### ISSUE THREAD

**Issue by TomTravaglino** [openhab/org.openhab.binding.zigbee#403]
With #342, in `ZigBeeBaseChannelConverter` the device initialization and converter initialization has been split into two methods, so that it becomes possible to only initialize the device, or the converter, or both.

This issue is for implementing the following extension suggested by @cdjackson in #338:
> Initially I would simply propose to call both methods during initialisation, however a further extension is to only call configure when the device is first discovered (need to think how this is detected) and it may also be useful to provide a user action to allow the user to manually re-configure the device (which would be especially useful for battery devices).

The basic idea is to introduce a boolean config parameter `zigbee_initialise_device` for things. The `ZigBeeThingHandler` will initialize the device only of this parameter is set to `true`, and it will set the parameter to `false` afterwards. The parameter will be set to `true` by the discovery service, so that the device is always initialized after discovery. Users can manually re-configure the device by updating the value of the parameter to `true`.

**Comment by TomTravaglino** [comment]
We started working on a PR

**Comment by TomTravaglino** [comment]
I ran into an issue with the approach described above. As described I set the config parameter to `true` in the discovery service and then to `false` when the device initialization is done. But when I restart the binding (I mean the bundles) the discovery starts automatically and discovers the same device again and then sets the config parameter of the existing thing to `true` again. This leads the thing handler to re-execute the device initialization.

My proposal is to implement this solution:
- Introduce a new thing property (e.g. 'zigbee_initialised') which will be set to `true` when the device initialization is done. When this parameter is set to `true` the thing handler will skip the device initialization.
- Add a config parameter `zigbee_initialise_device` to the thing. This parameter can be set by the user to manually initialize the device.
